src/helpers/deserializer.test.ts

```diff
@@ -9,53 +13,156 @@ describe('Deserializer helper', () => {
   }
 
   describe('deserialize', () => {
-    const jsonData = {
-      id: '1',
+    const jsonName = faker.person.fullName();
+    const jsonAge = faker.number.int();
+    const jsonDataFabricator = Fabricator({
+      id: () => faker.string.uuid(),
       type: 'TestType',
       attributes: {
-        name: 'Name',
-        age: 25,
+        name: jsonName,
+        age: jsonAge,
       },
-    };
-
-    it('returns deserialized data', () => {
-      const deserializedData = deserialize<TestType>(jsonData);
-
-      expect(deserializedData.resourceType).toBe('TestType');
-      expect(deserializedData.name).toBe('Name');
-      expect(deserializedData.age).toBe(25);
     });
-  });
 
-  describe('deserializeList', () => {
-    const jsonArray = [
-      {
-        id: '1',
-        type: 'TestType',
-        attributes: {
-          name: 'Name',
-          age: 25,
+    const questionId = faker.string.uuid();
+
+    const jsonDataWithRelation: Deserializer = {
```

Comment:
those `fabricators` should be moved out in a separate helper for re-use purposes.
Besides, you would consider DRY those fabricators

Reply:
@nvminhtue thanks, I fixed it in [5959b49](https://github.com/manh-t/react-survey/pull/69/commits/5959b49f57f9d5752d9030a5822af13dda119ea6)